src/engines/html5/html5.js

```diff
@@ -541,10 +543,6 @@ export default class Html5 extends FakeEventTarget implements IEngine {
             )
           );
         });
-      } else if (typeof this._el.webkitSetPresentationMode === 'function') {
-        this._el.webkitSetPresentationMode('inline');
-        // Safari does not fire this event but Chrome does, normalizing the behaviour
-        setTimeout(() => this.dispatchEvent(new FakeEvent(Html5EventType.LEAVE_PICTURE_IN_PICTURE)), 0);
```

Comment:
we don't need the event anymore?

Reply:
Actually, I don't see where this code was relevant, in all safari I've checked (mac, iPad with ios 11, 12, 13, and ios 14) this code was unreachable (means the `if` was applied not the `else if`). 
After this change I see this event fired natively.